gazebo_plugins/src/gazebo_ros_utils.cpp

```diff
@@ -108,22 +108,22 @@ void GazeboRos ::getParameterBoolean(bool &_value, const char *_tag_name) {
 
     if (sdf_->HasElement(_tag_name)) {
         std::string value = sdf_->GetElement(_tag_name)->Get<std::string>();
-        if(boost::iequals(value, std::string("true")))
+        if(boost::iequals(value, std::string("true")) || boost::iequals(value, std::string("1")))
         {
             _value = true;
         }
-        else if(boost::iequals(value, std::string("false")))
+        else if(boost::iequals(value, std::string("false")) || boost::iequals(value, std::string("0")))
         {
             _value = false;
         }
         else
         {
-            ROS_WARN_NAMED("utils", "%s: <%s> must be either true or false",
-                     info(), _tag_name);
+            ROS_WARN_NAMED("utils", "%s: <%s> must be either true or false but is '%s'",
```

Comment:
Warning should be updated to reflect new options
```"%s: <%s> must be either true/1 or false/0 but is '%s'"```

Reply:
Just to make it clear: My intention with this PR is not to introduce a new option, but to introduce the new behavior of the Gazebo API (melodic/gazebo9) to the evaluation of boolean strings in sdf. 'true'/'false' in an sdf are evaluated as '1'/'0' by the function `sdf_->GetElement(_tag_name)->Get<std::string>();`.
Just try yourself by e.g. setting up the diffkin plugin. You cannot configure it anymore without my PR.